crates/bitwarden-core/Cargo.toml

```diff
@@ -49,6 +50,7 @@ tsify-next = { workspace = true, optional = true }
 uniffi = { workspace = true, optional = true, features = ["tokio"] }
 uuid = { workspace = true }
 wasm-bindgen = { workspace = true, optional = true }
+wasm-bindgen-futures = { workspace = true, optional = true }
```

Comment:
This dep is used internally for the wasm-bindgen macro as cryptoclient has some async functions.